pkg/ast/ast.go

```diff
@@ -1584,9 +1584,10 @@ type AlterColumn struct {
 	Alter token.Pos // position of "ALTER" keyword
 	Null  token.Pos // position of "NULL"
 
-	Name    *Ident
-	Type    SchemaType
-	NotNull bool
+	Name        *Ident
+	Type        SchemaType
+	NotNull     bool
+	DefaultExpr *ColumnDefaultExpr
```

Comment:
Please update the above doc comment, and `End` method.

Reply:
Sure, my bad.